src/renderer/macos.css

```diff
@@ -0,0 +1,49 @@
+html[data-vcd-platform="macos"] .wrapper-1_HaEi {
```

Comment:
is having our own styles even necessary? Discord already has their own platform class they add to the html element, which is for example `platform-win` on windows

Reply:
![image](https://github.com/Vencord/Desktop/assets/45497981/874c8df5-461e-4e6a-a38a-ec566a645e45)

Reply:
yeah, the platform-osx class has almost the same styles as you attached